### ISSUE TITLE
Reference issues and tasks

### ISSUE THREAD

**Issue by ErMaqui** [MyCollab/mycollab#39]
A good feature can be to reference issues and tasks between them. For example, a issue can be blocked by another issue for resolving it.

Also, this can be a good option for creating a task with a list of issues / tasks to be resolved to complete it.

**Comment by haiphucnguyen** [comment]
It is a good idea! We are thinking to combine the meaning of tasks, issues and distinguish them by type, it will help we can multiple configure types and support the dependencies like what this issue describes.

**Comment by haiphucnguyen** [comment]
The latest MyCollab release 7.0.2 has this feature!
